content/en/continuous_integration/setup_pipelines/gitlab.md

```diff
@@ -10,6 +10,17 @@ further_reading:
       text: "Troubleshooting CI"
 ---
 
+## Compatibility
+
+Supported GitLab versions:
+* GitLab >= 14.1
+
+We also support the following versions with additional configuration:
+* GitLab >= 13.7.0, by enabling the `datadog_ci_integration` feature flag
+
+GitLab versions prior to 13.7.0 are supported using the [webhooks integration](#integrating-through-webhooks).
+
+## Enabling the Datadog integration
 
 To enable traces for your GitLab CI pipelines a [native integration][1] is available and hidden under a [feature flag][2] since release `13.7.0`.
```

Comment:
Any reason the "native integration" link doesn't take you to the Datadog specific page (https://docs.gitlab.com/ee/integration/datadog.html) vs the general one (https://docs.gitlab.com/ee/integration/)?

Reply:
More feedback, which doesn't relate to this PR specifically but I think is valuable either way:

This page mentions that the integration needs to be enabled per project. The GitLab page for the Datadog integration says it can be enabled per project or globally, but there aren't any instruction anywhere on how to enable it globally:

> You can also activate the integration for the entire GitLab instance.

Maybe it is obvious from the GitLab UI (I haven't used it), but it might be worth mentioning how.

Reply:
I refactored a bit the instructions, because they were out of date. I added how to enable it at the instance level as well. Please have another look.